src/net.ts

```diff
@@ -30,29 +44,47 @@ export default class Network {
     });
   }
 
+  checkProtocolVersion(): Promise<any> {
+    return new Promise(async (resolve, reject) => {
+      const response = await this.provider.send('ain_checkProtocolVersion',
+          '', { version: this.protoVer });
```

Comment:
Could it be skipped to pass the last parameter?

Reply:
Will change the api code first to look for protoVer instead of version, and then update this code afterwards. (Same for the other comment)